packages/experimental/Avatar/src/utils/getInitials-test.ts

```diff
@@ -0,0 +1,154 @@
+import { getInitials } from './getInitials';
+
+describe('getInitials', () => {
```

Comment:
I don't think this file gets run by Jest the way, is that by design?

Reply:
It runs. I'll rename it back. It was for investigating what causes build fails. The problem was with snapshot tests